### PR TITLE
chore: Enable Gremlin metric print on the console for every minute

### PR DESCRIPTION
**Pull request by arajkumar**
Reference: https://tinkerpop.apache.org/docs/current/reference/
Sample print:
```
-- Timers ----------------------------------------------------------------------
org.apache.tinkerpop.gremlin.server.GremlinServer.op.eval
             count = 6
         mean rate = 0.02 calls/second
     1-minute rate = 0.01 calls/second
     5-minute rate = 0.01 calls/second
    15-minute rate = 0.01 calls/second
               min = 10684.11 milliseconds
               max = 98444.63 milliseconds
              mean = 54317.79 milliseconds
            stddev = 46239.17 milliseconds
            median = 54044.93 milliseconds
              75% <= 97078.76 milliseconds
              95% <= 98444.63 milliseconds
              98% <= 98444.63 milliseconds
              99% <= 98444.63 milliseconds
            99.9% <= 98444.63 milliseconds
```

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -190,3 +190,9 @@ parameters:
   required: false
   name: JAVA_OPTIONS
   value: ""
+
+- description: Gremlin pool size
```
Comment (prashbnair) [inline openshift/template.yaml:194]: Why is this change required? The parameter is not used anywhere.

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: openshift/template.yaml (hunk 1 above)
Comment (arajkumar) [reply openshift/template.yaml:194]: It is used to control the gremlinPool.
https://github.com/fabric8-analytics/gremlin-docker/blob/master/scripts/entrypoint.sh#L56

We can override from saas templates to check whether it has any effect on the gremlin query delay.

Comment (prashbnair) [reply openshift/template.yaml:194]: My suggestion is to leave it as 8 as of now. Until we understand it completely lets not change it. I am fine with adding this for flexibility.

**Review by prashbnair — APPROVED** [review]
